includes/class-wc-install.php

```diff
@@ -248,6 +260,11 @@ public static function install_actions() {
 			check_admin_referer( 'wc_db_update', 'wc_db_update_nonce' );
 			self::update();
 			WC_Admin_Notices::add_notice( 'update' );
+			if ( WC()->is_wc_admin_active() ) {
```

Comment:
may be also add a check for if wc_admin is installed as well?

Reply:
I might be missing something, but it seems to me that if it's not installed, then it's not active (the method checks whether a function exists, so it only returns true if installed && active)?

Reply:
I think I meant we can check here if tables are also installed. Let me actually test this scenario and the one described in https://github.com/woocommerce/woocommerce/pull/25736#discussion_r383253956

Reply:
Resolving based on testing.